source/docs/tuner/device-history.rst

```diff
@@ -0,0 +1,26 @@
+Device History
+==============
+
+Users can access a list of past devices connected to Tuner X and license them via the :guilabel:`Device History` page. This is accessible from the left-hand sidebar. This list is not automatically refreshed, but users can refresh it by refreshing the page with the refresh icon in the top-right of the page.
```

Comment:
"but users can refresh it by ~~refreshing the page with~~ **pressing** the refresh icon in the top-right of the page."